### PR TITLE
CPDTP-482: Rejig the payment breakdown page

### PR DESCRIPTION
**Pull request by kicferk1**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDTP-482

### Is there anything that the code reviewer should know?
I opted to reuse the existing components. Not sure if calling them `row`s makes sense - any better names?

You might also spot 'TODO's in plenty of them - because the payment breakdown page from the prototype surfaces a bunch of things that we don't currently have. We will need to remember to add them. 

### Code quality checks
- [x] All commit messages are meaningful and true
- [x] Added enough automated tests

### HTML Checks
- [x] All new pages have automated accessibility checks
- [x] All new pages have visual tests via Percy

### Gotchas
- [x] All `School` queries are correctly scoped - `eligible` when they need to be

### How can someone see it it review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Sign in as `finance@example.com`
3. Go to payment breakdown page for a provider.

## External API changes

Does this change anything in our external APIs? If so, did you remember to update the CHANGELOG for Lead Providers? (lead-providers/guidance/release-notes) - Nope

### CODE REVIEW

File: app/components/finance/ecf/breakdown_summary.html.erb [hunk 1]
```diff
@@ -1,62 +1,108 @@
...
-      <%= participants %>
-    </dd>
-  </div>
-</dl>
+<%= @breakdown %>
+<table class="govuk-table ">
+  <thead>
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header"></th>
+      <th class="govuk-table__header"></th>
```
[the remark this answers is not in the document]
Comment (kicferk1) [reply app/components/finance/ecf/breakdown_summary.html.erb:6]: I think they give a fancy line above of the top row. I guess I could see if I can get the same effect with some css classes.

File: app/components/finance/ecf/breakdown_summary.html.erb [hunk 2]
```diff
@@ -1,62 +1,108 @@
...
-  <div class="govuk-summary-list__row">
-    <dt class="govuk-summary-list__key">
-      <%=t("finance.participants") %>
-    </dt>
-    <dd class="govuk-summary-list__value">
-      <%= participants %>
-    </dd>
-  </div>
-</dl>
+<%= @breakdown %>
```
[the remark this answers is not in the document]
Comment (kicferk1) [reply app/components/finance/ecf/breakdown_summary.html.erb:1]: Oooops, good spot!

**Comment by kicferk1** [comment]
> would be nice if the percy snapshot had the sections expanded. Probably more work than it's worth though

Fingers crossed it's as easy as clicking "open all" on the accordion

File: app/components/finance/ecf/breakdown_summary.html.erb [hunk 3]
```diff
@@ -1,62 +1,108 @@
...
+  <thead>
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header"></th>
+      <th class="govuk-table__header"></th>
+    </tr>
+  </thead>
+  <tbody>
+    <!--TODO: Un-hard-ode it-->
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">Payment period</td>
```
[the remark this answers is not in the document]
Comment (ethanmills) [reply app/components/finance/ecf/breakdown_summary.html.erb:5]: All of those things look like descriptions of what is in the row (not column) to me. Might be worth getting Stuart to take a quick look at this with a screen  reader if you're not sure they should be row scoped headings?

[the remark this answers is not in the document]
Comment (kicferk1) [reply app/components/finance/ecf/breakdown_summary.html.erb]: Govuk tables believe that headers should be bold by default, but eh, nothing a class can't fix. Should be ths now.

**Review by ethanmills — APPROVED** [review]

**Pull request merged**